.travis.yml

```diff
@@ -4,9 +4,10 @@ env:
 language: python
 python:
 - '2.7'
```

Comment:
ezt is el lehet engedni, korábban már volt erről szó

Reply:
A fork-olt repóból feladtam rá PR-t : https://github.com/shiwaforce/poco/pull/74